src/consensus/pbft/pbft.h

```diff
@@ -341,11 +347,13 @@ namespace pbft
         if (*gb_info->last_commit_view < view)
         {
           gb_info->view_change_list->emplace_back(view, version);
+          gb_info->encryptor->set_view(view);
```

Comment:
This is not correct, we need to set the view on the encryptor right after we view change. Will change this.